netket/experimental/sampler/metropolis_pt.py

```diff
@@ -380,30 +380,6 @@ def cb(data):
 
         return new_state, new_state.σ[new_state.beta_0_index + offsets, :]
 
-    def __repr__(sampler):
```

Comment:
why are you removing this?

Reply:
Because they did not really override anything. Previously we did not rename `reset_chain` to `reset_chains` in them, so they would cause errors.

Reply:
ah because it's a data class?